cpp/source/deal.II/geometry.templates.h

```diff
@@ -293,6 +330,20 @@ Geometry<dim>::Geometry(std::shared_ptr<boost::property_tree::ptree> database,
     }
     mesh_generator(*database);
   }
+
+  // We need to do load balancing because cells in the collectors and the
+  // separator don't have both physics.
+  std::array<unsigned int, 4> weights;
+  weights[internal::WEIGHT_TYPE::anode] = database->get("anode.weight", 0);
+  weights[internal::WEIGHT_TYPE::cathode] = database->get("cathode.weight", 0);
+  weights[internal::WEIGHT_TYPE::separator] =
+      database->get("separator.weight", 0);
+  weights[internal::WEIGHT_TYPE::collector] =
+      database->get("collector.weight", 0);
```

Comment:
Yes, there is a default value of 1000 that is added to the weight that we give. So if we give a weight of 1000 to a cell, its weight is doubled.